include/nbl/builtin/hlsl/scan/declarations.hlsl

```diff
@@ -9,8 +9,8 @@
 
 #include "nbl/builtin/hlsl/cpp_compat.hlsl"
 
-#ifndef NBL_BUILTIN_MAX_SCAN_LEVELS
-#define NBL_BUILTIN_MAX_SCAN_LEVELS 7
+#ifndef NBL_BUILTIN_MAX_LEVELS
+#define NBL_BUILTIN_MAX_LEVELS 7
```

Comment:
this should be a `NBL_CONSTEXPR_STATIC_INLINE` in `Parameters_t`

Reply:
I keep doing the same reviews
https://github.com/Devsh-Graphics-Programming/Nabla/pull/665#discussion_r1547511769